app/resources/v1/library_type_resource.rb

```diff
@@ -1,8 +1,31 @@
 # frozen_string_literal: true
 
 module V1
-  # Exposes valid library type options for use by the UI
+  # @todo This documentation does not yet include a detailed description of what this resource represents.
+  # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+  # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+  #
+  # @note Access this resource via the `/api/v1/library_type` endpoint.
+  #
+  # Provides a JSON:API representation of {LibraryType} and exposes valid library type options
+  # for use by the UI.
+  #
+  # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
+  # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package for the service
+  # implementation of the JSON:API standard.
   class LibraryTypeResource < JSONAPI::Resource
+    # @!attribute [rw] name
+    #   @return [String] the name of the library type
+    # @!attribute [rw] pipeline
+    #   @return [String] the pipeline associated with the library type
+    # @!attribute [rw] created_at
+    #   @return [DateTime] the timestamp when the library type was created
+    # @!attribute [rw] updated_at
+    #   @return [DateTime] the timestamp when the library type was last updated
```

Comment:
In theory these data times are strings in the json response. Im not sure which one is correct to use. In some other resources it is set to String and some DateTime so it would be good to make them consistent as well.